src/resources/Groups/GroupsInterfaces.ts

```diff
@@ -0,0 +1,93 @@
+export interface GroupModel {
+    id: string;
```

Comment:
At creation, if id is given it will throw.

Reply:
If you look at the `create` method on the group resource, we use the type `New<GroupModel>` which omits the `id` field.